### PR TITLE
feat: add the `eslint-visitor-keys` package

### PR DESCRIPTION
**Pull request by snitin315**
Refers https://github.com/eslint/espree/issues/609

### CODE REVIEW

File: .github/workflows/release-please.yml [hunk 1]
```diff
@@ -98,3 +98,29 @@ jobs:
             env:
                 MASTODON_ACCESS_TOKEN: ${{ secrets.MASTODON_ACCESS_TOKEN }}
                 MASTODON_HOST: ${{ secrets.MASTODON_HOST }}
+
+          #-----------------------------------------------------------------------------
```
Comment (nzakas) [inline .github/workflows/release-please.yml:102]: This needs to go first because Espree depends on `eslint-visitor-keys`

File: eslint.config.js [hunk 2]
```diff
@@ -15,7 +15,7 @@ export default [
     ...eslintConfigESLint,
     eslintConfigESLintFormatting,
     {
-        files: ["packages/espree/tests/lib/**"],
+        files: ["packages/espree/tests/lib/**", "packages/eslint-visitor-keys/tests/lib/**"],
```
Comment (nzakas) [inline eslint.config.js:18]: Maybe we can simplify?
```suggestion
        files: ["packages/*/tests/lib/**"],
```

Comment (snitin315) [reply eslint.config.js]: Yes, updated.

File: .github/workflows/release-please.yml (hunk 1 above)
Comment (snitin315) [reply .github/workflows/release-please.yml]: Updated 👍🏻

**Review by aladdin-add — APPROVED** [review]
LGTM. Would like @nzakas to verify before merging.

**Review by nzakas — APPROVED** [review]
LGTM. Thanks!

**Review by Playaset — COMMENTED** [review]
5903929cb65898ffdb6bea7d3fca99781f32ff09

**Pull request merged**